src/capi.cpp

```diff
@@ -1276,6 +1292,15 @@ int rtcSetAACPacketizationHandler(int tr, const rtcPacketizationHandlerInit *ini
 	});
 }
 
+int rtcSetRtcpReceivingSession(int tr) {
+	return wrap([&] {
+		auto track = getTrack(tr);
+		auto session = std::make_shared<rtc::RtcpReceivingSession>();
+		track->setMediaHandler(session);
```

Comment:
I could be good to call `chainMediaHandler` here instead so the `RtcpReceivingSession` can be used with other media handlers, for instance a packetizer/depacketizer.

Reply:
```suggestion
		track->chainMediaHandler(session);
```